decidim-conferences/app/views/decidim/conferences/admin/partners/index.html.erb

```diff
@@ -8,45 +8,64 @@
+          <th><%= t("models.partner.fields.actions", scope: "decidim.admin") %></th>
         </tr>
       </thead>
       <tbody>
         <% @partners.each do |partner| %>
           <tr>
-            <td>
+            <td data-label="<%= t("models.partner.fields.name", scope: "decidim.admin") %>">
               <%= partner.name %>
             </td>
-            <td>
+            <td data-label="<%= t("models.partner.fields.partner_type", scope: "decidim.admin") %>">
               <%= t("#{partner.partner_type}", scope: "decidim.admin.models.partner.types") %>
             </td>
-            <td>
+            <td data-label="<%= t("models.partner.fields.link", scope: "decidim.admin") %>">
               <% if partner.link.presence %>
                 <%= link_to partner.link, partner.link, target: "_blank" %>
               <% end %>
             </td>
-            <td>
+            <td data-label="<%= t("models.partner.fields.logo", scope: "decidim.admin") %>">
               <% if partner.logo.attached? %>
                 <%= image_tag(partner.attached_uploader(:logo).variant_url(:thumb)) %>
               <% end %>
             </td>
-            <td class="table-list__actions">
-              <% if allowed_to? :update, :partner, partner: partner %>
-                <%= icon_link_to "pencil-line", edit_conference_partner_path(current_conference, partner), t("actions.edit", scope: "decidim.admin"), class: "action-icon--edit" %>
-              <% end %>
+            <td class="table-list__actions" data-label="<%= t("models.partner.fields.actions", scope: "decidim.admin") %>">
+              <button type="button" data-component="dropdown" data-target="actions-partner-<%= partner.id %>" aria-label="<%= t("decidim.admin.actions.actions_label", resource: partner.name || t("models.partner.name", scope: "decidim.admin")) %>">
+                <%= icon "more-fill", class: "text-secondary" %>
+              </button>
 
-              <% if allowed_to? :destroy, :partner, partner: partner %>
-                <%= icon_link_to "delete-bin-line", conference_partner_path(current_conference, partner), t("actions.destroy", scope: "decidim.admin"), class: "action-icon--remove", method: :delete, data: { confirm: t("actions.confirm_destroy", scope: "decidim.admin") } %>
-              <% end %>
+              <div class="inline-block relative">
+                <ul id="actions-partner-<%= partner.id %>" class="dropdown dropdown__action" aria-hidden="true">
+                  <% if allowed_to? :update, :partner, partner: partner %>
+                    <li class="dropdown__item">
+                      <%= link_to edit_conference_partner_path(current_conference, partner), class: "dropdown__button" do %>
+                        <%= icon "pencil-line" %>
+                        <%= t("actions.edit", scope: "decidim.admin") %>
+                      <% end %>
+                    </li>
+                  <% end %>
+
+                  <hr>
```

Comment:
Let's move the HR inside the above permission check

Reply:
Fixed on 2edcdcb5ea1d34de68fb35c5d3038169988dc759